docs/DevelopmentSetup/visual_studio_win_example.rst

```diff
@@ -38,6 +45,13 @@ Launch the Windows Powershell and run the following commands:
     mkdir -p Source\build
     cd Source\build
 
+.. figure:: ../images/installation_vs2019_flint.example/Step2.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
```

Comment:
Alt-text are important for accesibility (e.g. https://bitsofco.de/the-accessibility-cheatsheet/). Could you please add a short summary to these images?

Reply:
Hey @aornugent Yes I agree I was thinking of adding them all at once so that I don't miss any :v: otherwise we can update this one as well